packages/ontario-design-system-component-library/src/components/ontario-task/ontario-task.tsx

```diff
@@ -0,0 +1,221 @@
+import { h, Component, Prop, Watch, State, Listen } from '@stencil/core';
+import { ConsoleMessageClass } from '../../utils/console-message/console-message';
+import { TaskStatuses, TaskStatus, BadgeColors } from './ontario-task-statuses';
+import { validateLanguage, validateValueAgainstArray } from '../../utils/validation/validation-functions';
+import { Hint } from '../../utils/common/common.interface';
+import { Language } from '../../utils/common/language-types';
+import { constructHintTextObject } from '../../utils/components/hints/hints';
+import translations from '../../translations/global.i18n.json';
+
+@Component({
+	tag: 'ontario-task',
+	styleUrl: 'ontario-task.scss',
+	shadow: true,
+})
+export class OntarioTask {
+	hintTextRef: HTMLOntarioHintTextElement | undefined;
+
+	/**
+	 * Specifies the label of the task.
+	 *
+	 * This is required to provide the name of the task.
+	 */
+	@Prop() label: string;
+
+	/**
+	 * Specifies an optional link associated with the task.
+	 *
+	 * If provided, clicking the task will navigate to this URL.
+	 */
+	@Prop() link?: string;
+
+	/**
+	 * Disables the task link when set to `true`.
+	 *
+	 * Default is `false`, meaning the link will be active if provided.
+	 */
+	@Prop() deactivateLink: boolean = false;
```

Comment:
What's the use case for `deactivateLink`?

Reply:
This is if a user cannot yet start the task if they haven't completed a previous task.

Reply:
Or perhaps if the status is completed. It just gives them options.

Reply:
Negatives can be confusing and the use of a verb makes it sound like an action. 

```suggestion
	 *
	 * Default is `true`, meaning the link will be active if provided.
	 */
	@Prop() active: boolean = true;
```